python/manageJobs.py

```diff
@@ -224,6 +240,14 @@ def manageJobs(argv=None):
         parser.error("Options -h, -r, -i, -f are exclusive, pick one!")
     if options.resubmit and options.kill:
         parser.error("Can't use -s and -k together, pick one!")
+    if options.xrootdResubmit and options.kill:
+        parser.error("Can't use -X and -k together, pick one!")
+    if options.inputFileClassAd and options.logKey:
+        parser.error("Can't use -C and -L/-K together, pick one!")
+    if options.xrootdResubmit and not options.inputFileClassAd and not options.logKey:
+        parser.error("To gather the input file names, you must specify either a classad (-C) or the path (-L) to some logs and the key to parse them (-K), choose your method!")
+    if options.xrootdResubmit and options.logKey and not options.logPath:
+        parser.error("When speficying a key for log parsing you must also specify the path to the logs (-L)")

```

Comment:
speficying -> specifying

Reply:
should you also check for the other case (path but no key?)

Reply:
Path has a default value (`os.environ["PWD"]`) so that makes it hard to have a key but no path. The user would have to do that intentionally. Therefore I didn't test for that case. Anyway, if that did happen, then the issue would be caught by file_finder.py.

Reply:
this line does test for key and no path...

Reply:
i think this is fixed now.